pkg/registry/util.go

```diff
@@ -129,3 +135,80 @@ func parseReference(raw string) (registry.Reference, error) {
 
 	return registry.ParseReference(raw)
 }
+
+// generateOCIAnnotations will generate OCI annotations to include within the OCI manifest
+func generateOCIAnnotations(meta *chart.Metadata) map[string]string {
+
+	// Get annotations from Chart attributes
+	ociAnnotations := generateChartOCIAnnotations(meta)
+
+	// Copy Chart annotations
+annotations:
+	for chartAnnotationKey, chartAnnotationValue := range meta.Annotations {
+
+		// Avoid overriding key properties
+		for _, immutableOciKey := range immutableOciAnnotations {
+			if immutableOciKey == chartAnnotationKey {
+				continue annotations
+			}
+		}
+
```

Comment:
should this be indented in?

Reply:
Labels are not indented: https://go.dev/doc/effective_go#switch